rdt/transformers/categorical.py

```diff
@@ -329,4 +329,4 @@ def reverse_transform(self, data):
         if isinstance(data, np.ndarray) and (data.ndim == 2):
             data = data[:, 0]
 
-        return pd.Series(data).astype(int).map(self.values_to_categories)
+        return pd.Series(data).map(self.values_to_categories)
```

Comment:
Is this change right, considering that input could be float values too?
On the other side, maybe the `astype(int)` should rather be `round().astype(int)` instead?

Reply:
Can the input be floats? `transform` should only yield `int` values right? Also replacing with `round().astype(int)` shouldn't help since it's the `astype` part that is slow. However maybe we can just `round` since if you have `d={1.0: 'a'}` and then do `d[1]` you will still get the value

Reply:
Yes, reverse transform should always work properly with float inputs, since those are the values outputted by the underlying models.
On the other side, I agree that the `astype` may not be needed, so we can possibly do `round` only.